test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -934,78 +940,102 @@ TEST_F(ClusterManagerImplTest, InitializeOrder) {
  version_info: version3
  static_clusters:
   - cluster:
-      "@type": type.googleapis.com/envoy.api.v2.Cluster
+      "@type": type.googleapis.com/envoy.api.v3.Cluster
```

Comment:
CI says the [cluster_manager_impl_test failed](https://dev.azure.com/cncf/envoy/_build/results?buildId=38618&view=logs&j=8d802004-fbbb-5f17-b73e-f23de0c1dec8&t=86512d3a-caa5-5e3f-9bce-4070faa3b830&l=5459) with `...invalid value Invalid type URL, unknown type: envoy.api.v3.Cluster for type Any...`

That is because the cluster message is now at: [`envoy.config.cluster.v3.Cluster`](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/cluster/v3/cluster.proto#L35). The v2 proto provides a hint [here](https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/cluster.proto#L30).

Reply:
@junr03 thanks.